### PR TITLE
Use 'dosome.click' for Bertly 2.0

### PR DESCRIPTION
**Pull request by DFurnes**
### What's this PR do?

This pull request updates the domain mapping for Bertly 2.0 to use dosome.click.

### How should this be reviewed?

:eyes:

### Any background context you want to provide?

It's launch day! :rocket: 

### Relevant tickets

References [Pivotal #172652963](https://www.pivotaltracker.com/story/show/172652963).

### Checklist

- [ ] This PR has been added to the relevant Pivotal card.

### CODE REVIEW

File: dosomething/main.tf [hunk 1]
```diff
@@ -69,8 +69,8 @@ module "bertly" {
 
   environment   = "production"
   name          = "dosomething-bertly"
-  domain        = "new.dosome.click"
-  certificate   = "*.dosome.click"
+  domain        = "dosome.click"
+  certificate   = "dosome.click"
```
Comment (katiecrane) [inline dosomething/main.tf:73]: Is the old Bertly still somewhere using `dosome.click`?

Comment (DFurnes) [reply dosomething/main.tf:73]: Yep! I'm going to detach that mapping before I apply this. :link:

Comment (katiecrane) [reply dosomething/main.tf:73]: Ah so that one is outside of terraform?

Comment (katiecrane) [reply dosomething/main.tf:73]: Sounds good though - now I'm just curious!

Comment (DFurnes) [reply dosomething/main.tf:73]: Yes! Originally, Bertly was managed with [CloudFormation](https://aws.amazon.com/cloudformation/), Amazon's configuration-as-code tool. It had some nice integrations in the AWS web GUI, but also only worked with AWS services which was a deal-breaker for a lot of our stuff.

**Pull request merged**